fixtures.ttar

```diff
@@ -6551,3 +7047,7 @@ Lines: 1
 4733
 Mode: 644
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Path: fixtures/.unpacked
+Lines: 0
+Mode: 644
+# ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
This shouldn't be in here. There's a `make update_fixtures` helper that takes care of re-packing the fixtures.

Reply:
make update_fixtures done